travis-docker-compose.yml

```diff
@@ -31,31 +31,25 @@ services:
       BROKER_URL: redis://redis:6379/4
       CELERY_RESULT_BACKEND: redis://redis:6379/4
     env_file: .env
+
+  web:
+    image: micromasters_python
```

Comment:
Yes, the image is specified here so we don't go through the docker build multiple times